homeassistant/components/device_tracker/mikrotik.py

```diff
@@ -68,13 +70,26 @@ def connect_to_device(self):
         """Connect to Mikrotik method."""
         import librouteros
         try:
-            self.client = librouteros.connect(
-                self.host,
-                self.username,
-                self.password,
-                port=int(self.port),
-                encoding='utf-8'
-            )
+            if self.ssl:
+                ssl_context = ssl.create_default_context()
```

Comment:
Where does `ssl` come from?

Reply:
Sorry I missed something. The fix is committed.